chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,88 @@
+import sys
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from poleno.utils.misc import squeeze
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    args = u'attachment_id [file]'
+    help = squeeze(u"""
+            Creates anonymization for the specified Attachment. The content source is file, that can
+            be passed as an argument, or stdin. Preferred source is file. If file is not specified
+            and stdin is empty, the command will fail. Anonymization created this way will be marked
+            as successful. Only one successful anonymization can be assigned to the Attachment.
```

Comment:
@viliambalaz Trochu som to preformuloval a zjednodusil. Spominat, ze anonymizacia je successful asi nemusime, kedze z pohladu usera unsuccessful anonymizacie su nepodstatne, ako keby ani neexistovali. Je to len interny technicky detail.

> Creates anonymization for the specified Attachment. The anonymized content is read from the given file or from stdin if no file is specified. If no file is specified and stdin is empty, the command will fail.

Reply:
:ok: